File: 1426-find-n-unique-integers-sum-up-to-zero/1426-find-n-unique-integers-sum-up-to-zero.cpp
class Solution {
public:
    vector<int> sumZero(int n) {
        vector<int> ans;
        int md = n / 2;
        for(int i=-1*md;i<=md;i++){
            if(n%2==0 && i == 0)continue;
            ans.push_back(i);
        }

        return ans;
    }
};

File: 1426-find-n-unique-integers-sum-up-to-zero/1426-find-n-unique-integers-sum-up-to-zero.cs
public class Solution {
    public int[] SumZero(int n) {
        List<int> ans = new List<int>();
        int md = n/2;
        for(int i=-md;i<=md;i++){
            if(n%2==0 && i == 0)continue;
            ans.Add(i);
        }
        return ans.ToArray();
    }
}